### DISCUSSION TITLE
Multiple deployment jobs for single environment

### DISCUSSION THREAD

**Discussion by ABAG603** [community/community#68403] [Actions]
Product Feedback

Hi,

is there any way or is it planned to add such functionality as multiple deployment jobs for single environment? My scenario is: when deployment starts and approved, first job that has to run is DB Update, after it completes I need run start several jobs in parallel to deploy services. I can add "environment:" to every job but then I'll have to approve multiple times. What I need is to approve once and all jobs with same environment to be executed as defined in workflow.

**Comment by vishalkrishna8** [comment]
Hello @ABAG603,
GitHub Actions did not support running multiple deployment jobs for a single environment with a single approval. Each job could be associated with a different environment and might require separate approvals. 
Make my reply as an answer if you find it helpful. 
Thank you

**Comment by sullimander** [comment]
It would be really great if we could specify the environment at the top level of a workflow, allowing for the full workflow to be the deployment.
